api/SVGPointList.json

```diff
@@ -291,7 +291,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": true
+              "version_added": "79"
```

Comment:
This is a `length` property, and at first I though it's unlikely that `SVGPointList` was supported without this, but there's also `numberOfItems` which was supported from Edge 12. http://mdn-bcd-collector.appspot.com/tests/api/SVGPointList/length looks correct to me too.